documentation/collectionrules/TriggerShortcuts.md

```diff
@@ -0,0 +1,201 @@
+
+<details>
+  <summary>JSON</summary>
+
+  ```json
+  {
+    "Trigger": {
+      "Type": "CPUUsage"
+    }
+  }
+  ```
+</details>
+
+<details>
+  <summary>Environment Variables</summary>
+  
+  ```bash
+  export DotnetMonitor_CollectionRules__RuleName__Trigger__Type="CPUUsage"
+  ```
+</details>
+
+### `GCHeapSize` Trigger Shortcut
+
+Execute a trigger when the target application's GC Heap Size is continuously greater than or less than a specified value for a set duration of time.
+
+#### Properties
+
+| Name | Type | Required | Description | Default Value | Min Value | Max Value |
+|---|---|---|---|---|---|---|
+| `GreaterThan` | double? | false | The threshold level in MBs that the counter must maintain (or higher) for the specified duration. | `10` | `0` | |
+| `LessThan` | double? | false | The threshold level in MBs that the counter must maintain (or lower) for the specified duration. If `LessThan` is specified, the default value of `GreaterThan` becomes `null`. | `null` | `0` | `100` |
+| `SlidingWindowDuration` | false | TimeSpan? | The sliding time window in which the counter must maintain its value as specified by the threshold levels in `GreaterThan` and/or `LessThan`. | `"00:01:00"` (one minute) | `"00:00:01"` (one second) | `"1.00:00:00"` (1 day) |
+
+#### Equivalent EventCounter Trigger
+
+<details>
+  <summary>JSON</summary>
+
+  ```json
+  {
+    "Trigger": {
+      "Type": "EventCounter",
+      "Settings": {
+        "ProviderName": "System.Runtime",
+        "CounterName": "gc-heap-size",
+        "GreaterThan": "10"
+      }
+    }
+  }
+  ```
+</details>
+
+<details>
+  <summary>Environment Variables</summary>
+  
+  ```bash
+  export DotnetMonitor_CollectionRules__RuleName__Trigger__Type="EventCounter"
+  export DotnetMonitor_CollectionRules__RuleName__Trigger__Settings__ProviderName="System.Runtime"
+  export DotnetMonitor_CollectionRules__RuleName__Trigger__Settings__CounterName="gc-heap-size"
+  export DotnetMonitor_CollectionRules__RuleName__Trigger__Settings__GreaterThan="10"
+  ```
+</details>
+
+#### Example
+
+Usage that is satisfied when the GC Heap Size of the application is greater than 10MB for a 1 minute window.
+
+<details>
+  <summary>JSON</summary>
+
+  ```json
+  {
+    "Trigger": {
+      "Type": "GCHeapSize"
+    }
+  }
+  ```
+</details>
+
+<details>
+  <summary>Environment Variables</summary>
+  
+  ```bash
+  export DotnetMonitor_CollectionRules__RuleName__Trigger__Type="GCHeapSize"
+  ```
+</details>
+
+### `ThreadpoolQueueLength` Trigger Shortcut
+
+Execute a trigger when the target application's threadpool queue length is continuously greater than or less than a specified value for a set duration of time.
+
+#### Properties
+
+| Name | Type | Required | Description | Default Value | Min Value | Max Value |
+|---|---|---|---|---|---|---|
+| `GreaterThan` | double? | false | The threshold level the counter must maintain (or higher) for the specified duration. | `200` | `0` | |
```

Comment:
@poppastring  Has done more research here, but should we be supplying a default, given how workload dependent this value is?